DigitalExcellence/dex-backend
# Feature/377 fontys gitlab datasource to wizard

## Description

Added the functionality to search for repositories using the Fontys Gitlab, private projects for gitlab should also work. @rubenb994 can you maybe test if gitlab still works? I don't have the client secret etc...

## Type of change


## Checklist


## Steps to Test or Reproduce

1. Get a user token, clientid and clientsecret from the Fontys Gitlab (in your user settings).
2. Open swagger and check all 3 wizard endpoints using the token and other meta data specific to the request.
3. Check if gitlab works with private and public projects.

## Link to issue

Closes: #377
